index.bs

```diff
@@ -11302,6 +11306,153 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The extensions Module ## {#module-extensions}
+
+The <dfn export for=modules>extensions</dfn> module contains functionality for
+managing and interacting with extensions.
+
+### Definition ### {#module-extensions-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+ExtensionsCommand = (
+  extensions.Install
+)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+ExtensionsResult = (
+  extensions.InstallResult
+)
+</pre>
+
+### Types ### {#module-extensions-types}
+
+#### The extensions.Extension Type #### {#type-extensions-Extension}
+
+<pre class="cddl local-cddl">
+extensions.ExtensionArchive = {
+   files: [+text],
+}
+</pre>
+
+<pre class="cddl remote-cddl local-cddl">
+extensions.Extension = {
+   installPath: extensions.ExtensionPath / extensions.ExtensionArchivePath / extensions.ExtensionBase64Encoded,
+   ? allowPrivateBrowsing: bool .default false,
```

Comment:
@oliverdunk do you know if we have an equivalent of what allowPrivateBrowsing is expressing here?

Reply:
In Chromium this is called "Allow in Incognito"
<img width="753" alt="Screenshot 2024-10-09 at 21 17 41" src="https://github.com/user-attachments/assets/273580f3-b8b6-4863-a102-65688ec7c2d9">

Reply:
I see, thanks! I wonder if alternatively we could specify a user context ID to install an extension to? (I would need to check if it's implementable though)

Reply:
Actually my last answer was wrong. Temporary actually means that the web extension is not persistent and can be installed even when it isn't signed (like for developers), and that is indeed a feature that we support in Firefox. 

But we do not yet support the `private browsing` mode installation. So both things are different. 

@OrKoN How does Chrome handle the installation of unsigned web extensions?